assets/context.py

```diff
@@ -0,0 +1,187 @@
+import asyncio
+from collections import namedtuple
+
+import discord
+from discord.ext import commands
+
+from cogs.utils import misc, check
+from cogs.utils.embeds import ConfirmEmbed, ConfirmEmbedAnother
+
+argument = namedtuple("Usage", ["required", "text"])
+example = namedtuple("Examples", ["args", "text"])
+sub_desc = namedtuple("subcommands", "desc")
+others = namedtuple("others", ["name", "text"])
+usage_info = namedtuple("usage_information", ["command", "value"])
+keyword = namedtuple("keywords", ["keyword", "desc"])
+
+
+class InvokeError(commands.CommandError):
+    def __init__(self, message):
+        self.message = message
+
+
+class Context(commands.Context):
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+        self.InvokeError = InvokeError
+
+    async def confirm(self, title, content, color=misc.random_color()):
```

Comment:
C0116: Missing function or method dosctring

I'm not gonna add all of them : you're missing your docstrings